src/components/ImageUpload/ImageUploadButton.tsx

```diff
@@ -0,0 +1,47 @@
+import { useRef } from 'react';
+
+import Icon from '../Icon';
+
+interface Props {
+  setImageUrl: (imageUrl: string) => void;
+}
+
+const ImageUploadButton = ({ setImageUrl }: Props) => {
+  const inputElement = useRef<HTMLInputElement>(null);
+
+  const handleImageSelect = () => {
+    console.log(inputElement.current);
+    if (!inputElement.current) {
+      return;
+    }
+    if (!inputElement.current.files) {
+      return;
+    }
+    const [file] = inputElement.current.files;
+    if (!file) {
+      return;
+    }
+    const imageUrl = URL.createObjectURL(file);
+    setImageUrl(imageUrl);
+  };
+
+  return (
+    <div>
+      <label htmlFor='imageUploadButton'>
+        <div className='h-[35px] w-[35px] cursor-pointer rounded-full border border-gray-accent5 bg-white fill-gray-accent5 p-1.5'>
```

Comment:
`div.cursor-pointer`로 버튼을 잘 표현해주신 것 같아요 :)

`button`태그를 쓰시면 조금 더 시맨틱하면서 `cursor-pointer`를 뺄 수 있어서 더 간결해질 수도 있을 것 같아요 :)

Reply:
button으로 변경하니까 이미지업로드창이 안뜹니다!

Reply:
저도 해봤는데 안 돼서 찾아보니까 뭔가 이슈가 있는 것 같네요.. 

https://stackoverflow.com/questions/37504383/button-inside-a-label

button을 쓸 순 없을 것 같아요..

Reply:
이 부분은 공통 컴포넌트 `Button`을 쓰시면 될 것 같아요~

현재 카메라 아이콘이 중앙 정렬이 안 되어있는데, `Button` 컴포넌트를 사용하면 이 부분도 해결됩니다🙂

그리고 `icon-sprite` 파일에 `fill="currentColor"`가 추가되면서(#6) `fill-색`으로는 아이콘 색상이 변경되지 않고, `text-gray-accent5`와 같이 지정해주어야 스타일이 적용됩니다! 아래 suggestion 코드 참고해주세요 ㅎㅎ
```suggestion
        <Button className='h-[35px] w-[35px] rounded-full border border-gray-accent5 bg-white text-gray-accent5'>
```

Reply:
희라님 의견 감사합니다!! 적용하였습니다